### ISSUE TITLE
installing via .sbt/0.13/plugins causes ... weirdness

### ISSUE THREAD

**Issue by crispywalrus** [jrudolph/sbt-optimizer#7]
Even at the 0.1.0 stage this has some features I've been needing for a while. I was so enamored that I moved it to my .sbt plugins. Which caused a rather odd failure, all of a sudden ````sbt.Resolver.jcenterRepo```` became not-defined. Removing it resolved the issue but still it's something to think about.

**Comment by jrudolph** [comment]
That looks like an sbt version incompatibility. Though, I'm not sure how it is invoked exactly. What's the exact error you get and what sbt version does the problematic build use?

**Comment by crispywalrus** [comment]
Look through https://gist.github.com/crispywalrus/b203b0d282e540ab9251

if plugins.sbt is in $projectdir/project then everything works correctly but if plugins.sbt is in ````~/.sbt/0.13/plugins```` then I get an error
````
> Chris-Vale:adserver chris.vale$ sbt 
[info] Loading global plugins from /Users/chris.vale/.sbt/0.13/plugins
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[info] Loading project definition from /Users/chris.vale/adserver/project
/Users/chris.vale/adserver/build.sbt:7: error: value jcenterRepo is not a member of object sbt.Resolver
resolvers += Resolver.jcenterRepo
                      ^
[error] Type error in expression
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? 
````

**Comment by jrudolph** [comment]
For some reason sbt-optimizer 0.1.0 introduced a hard dependency on sbt-scripted 0.13.5 which in turn loads another version of sbt into the sbt classpath. This can lead to those strange issues as observed here.

I fixed this in 6e7ac5210950f292da44b89b73526ca99fd83f2b and made a new 0.1.1 release which is on the way into the central maven repo. Could you try if this fixes your issue once it turns up at http://repo1.maven.org/maven2/net/virtual-void/sbt-optimizer_2.10_0.13/?

Thanks again for the report.

**Comment by crispywalrus** [comment]
Thanks for this @jrudolph 0.1.1 does indeed fix the issue!
